specifications/transactions/offchain-api/README.md

```diff
@@ -0,0 +1,256 @@
+# Diem Off-Chain Protocol
+As defined in [DIP-1](https://dip.diem.com/dip-1), the Diem Off-Chain Protocol defines an API and payload specification to privately exchange information between two parties that cannot (easily) be achieved directly on a blockchain. This information includes the sensitive details of the parties involved in a payment that should remain off-chain. The information is exchanged within an authenticated and encrypted channel and only made available to the parties that directly involved.  Note that this API is being published by the Diem Association on an “as is” basis. Publication of this Off-Chain Protocol by the Diem Association does not mean that the Association is taking any position on whether the Off-Chain Protocol addresses issues of compliance, privacy or scalability. Users of this Off-Chain Protocol must make such determinations on their own.
```

Comment:
```suggestion
As defined in [DIP-1](https://dip.diem.com/dip-1), the Diem Off-Chain Protocol defines an API and payload specification to privately exchange information between two parties that cannot (easily) be achieved directly on a blockchain. This information includes the sensitive details of the parties involved in a payment that should remain off-chain. The information is exchanged within an authenticated and encrypted channel and only made available to the parties that are directly involved.  Note that this API is being published by the Diem Association on an “as is” basis. Publication of this Off-Chain Protocol by the Diem Association does not mean that the Association is taking any position on whether the Off-Chain Protocol addresses issues of compliance, privacy or scalability. Users of this Off-Chain Protocol must make such determinations on their own.
```

Reply:
> Publication of this Off-Chain Protocol by the Diem Association does not mean that the Association is taking any position on whether the Off-Chain Protocol addresses issues of compliance, privacy or scalability. Users of this Off-Chain Protocol must make such determinations on their own.

I really dislike that this sort of verbage is necessary. "Look we're making a protocol that if you want to interoperate with other players on the network you must use....but we absolve ourselves if it doesn't work"

Reply:
😢  -- I don't disagree, actually since this isn't the DIP, I am going to chop this language.